### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by Faalentijn**
Fixing the first item in my todo list in #338 which is to use code-blocks instead of literals to make it more readable. There are also a few more minor fixes which mainly revolve around making it easier to add it to the sphinx documentation.

### CODE REVIEW

**Comment by leandrotoledo** [comment]
Hi @Faalentijn, thanks for this PR!
Review is in progress...

File: docs/Makefile [hunk 1]
```diff
@@ -189,4 +189,4 @@ xml:
 pseudoxml:
 	$(SPHINXBUILD) -b pseudoxml $(ALLSPHINXOPTS) $(BUILDDIR)/pseudoxml
 	@echo
-	@echo "Build finished. The pseudo-XML files are in $(BUILDDIR)/pseudoxml."
+	@echo "Build finished. The pseudo-XML files are in $(BUILDDIR)/pseudoxml."
```
Comment (leandrotoledo) [inline docs/Makefile]: Any specific reasons for EOF without a newline?

**Comment by leandrotoledo** [comment]
One minor comment, everything else looks good.

File: docs/Makefile (hunk 1 above)
Comment (Faalentijn) [inline docs/Makefile]: I was asked to do it by @tsnoam in the [previous pull request](https://github.com/python-telegram-bot/python-telegram-bot/pull/337#issuecomment-231463983).

Comment (leandrotoledo) [inline docs/Makefile]: LGTM

**Comment by leandrotoledo** [comment]
Looks nicer, thanks @Faalentijn. Merging.

**Pull request merged**